.gitlab/kernel_version_testing/security_agent.yml

```diff
@@ -0,0 +1,266 @@
+# System probe overrides
+upload_dependencies_security_agent_x64:
+  extends:
+    - .package_dependencies
+  needs: ["pull_test_dockers_x64"]
+  rules: !reference [.on_security_agent_changes_or_manual]
+  variables:
+    ARCHIVE_NAME: dependencies-x86_64.tar.gz
+    ARCH: amd64
+    INSTANCE_TYPE: m5d.metal
+    TEST_COMPONENT: security-agent
+
+upload_dependencies_security_agent_arm64:
+  extends:
+    - .package_dependencies
+  needs: ["pull_test_dockers_arm64"]
+  rules: !reference [.on_security_agent_changes_or_manual]
+  variables:
+    ARCHIVE_NAME: dependencies-arm64.tar.gz
+    ARCH: arm64
+    INSTANCE_TYPE: m6gd.metal
+    TEST_COMPONENT: security-agent
+
+upload_minimized_btfs_security_agent_x64:
```

Comment:
We do not use minimized btfs in the CWS code, we manage our btfhub data in a different way